packages/core/utils/src/order/events.ts

```diff
@@ -0,0 +1,6 @@
+export const orderEvents = {
+  PLACED: "order.placed",
+  UPDATED: "order.updated",
+  CANCELED: "order.canceled",
+  COMPLETED: "order.completed",
+}
```

Comment:
suggestion: this does not follow the convention, I would suggest to lower case the keys and remove the updated
```
export const orderEvents = {
  places: "order.placed",
  canceled: "order.canceled",
  completed: "order.completed",
}
```

Reply:
I would maybe push to have us change the convention to use an uppercased version of the action:

So:
```
OrderEvents.PLACED
```
instead of 
```
OrderEvents.placed
```

Makes it more obvious that it is a "global constant".

Not necessarily right now, but thought I'd share my thinking.

Reply:
yes for sure, we just need to update the utils that build the constants 👍

Reply:
+1 for uppercase, a bit easier to read.